webapp/src/actions/index.js

```diff
@@ -80,4 +81,27 @@ export function performSearch(terms) {
 
         return dispatch(searchPostsWithParams(teamId, {terms, is_or_search: false, include_deleted_channels: viewArchivedChannels, page: 0, per_page: 20}, true));
     };
-}
\ No newline at end of file
+}
+
+export function requeueItem(itemId) {
+    return async (dispatch, getState) => {
+        const command = `/agenda requeue ${itemId}`;
```

Comment:
Is `itemId` a post id? If so, it seems like having this `requeue` slash command doesn't make much sense. 1/5 we should implement this as an HTTP endpoint instead, using the server-side [ServeHTTP](https://developers.mattermost.com/integrate/plugins/server/reference/#Hooks.ServeHTTP) hook.

Reply:
Right, the slash command makes sense then. Maybe we change this to `/agenda requeue post ${postId}`

Reply:
Copied over from https://github.com/mattermost/mattermost-plugin-agenda/pull/88#discussion_r704020987:

Made `/agenda requeue` to support future options like
`/agenda requeue all` # meeting canceled, I want to requeue all items to next meeting.
`/agenda requeue last` # requeue last items
`/agenda requeue last n` # requeue last n items